Assets/MRTK/Core/Inspectors/Setup/MixedRealityEditorSettings.cs

```diff
@@ -123,10 +126,16 @@ private static void LogConfigurationWarnings()
                     Debug.LogWarning("<b>Depth Buffer Sharing</b> has 24-bit depth format selected. Consider using 16-bit for performance. See <i>Mixed Reality Toolkit</i> > <i>Utilities</i> > <i>Optimize Window</i> tool for more information to improve performance");
                 }
 
-                if (!AudioSettings.GetSpatializerPluginName().Equals(MSFT_AudioSpatializerPlugin))
+                if (!UwpRecommendedAudioSpatializers.Contains<string>(SpatializerUtilities.CurrentSpatializer))
+                {
+                    Debug.LogWarning($"This application is not using the recommended <b>Audio Spatializer Plugin</b>. Go to <i>Project Settings</i> > <i>Audio</i> > <i>Spatializer Plugin</i> and select one of the following: {string.Join(", ", UwpRecommendedAudioSpatializers)}.");
+                }
+            }
+            else
```

Comment:
Change this from else -> if to else if (there's no need to have an else and then an if unless you have other clauses, and even if that were the case it would probably still be better to restructure things in that case)

Reply:
Updated